templates/articles.tpl

```diff
@@ -8,7 +8,13 @@
  * Deposit articles in remote repositories
  *}
 {include file="common/header.tpl" pageTitle="plugins.importexport.sword.displayName"}
- 
+
+<script type="text/javascript">
+	$(function() {ldelim}
+		$(document).data('depositedIdMap', {$depositedIds});
+	{rdelim});
+</script>
+<script src="{$pluginJavaScriptURL}/SwordDisableDepositedItems.js"></script>
```

Comment:
I think you can use TemplateManager::addJavaScript instead of `{$pluginJavaScriptURL}` and your own `<script>` elements.

Reply:
Done.